README.md

```diff
@@ -1 +1,25 @@
-# group-four-BE
\ No newline at end of file
+# group-four-BE
+## How to Make Contributions to this repo
+
+1. Fork and Clone the repo.
+2. Add the original repo as a remote called upstream.
+3. Create a new branch to work (e.g `git checkout -b ft-resoruces-rest-endpoint`).
+4. Implement your feature on the branch you created.
+5. Test your work and ensure everything is working fine.
+6. Pull the latest changes from upstream. (e.g. `git pull upstream develop`).
+5. Commit and push your changes to github.
+6. Create a pull request.
+
+### Branch Naming Conventions
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 0; Below](https://app.codacy.com/gh/Denniman/group-four-BE/pullRequest?prid=7777164)
```suggestion
### Branch Naming Conventions

```